backend/lib/watches/leases.js

```diff
@@ -44,8 +44,10 @@ module.exports = (io, informer, { signal }) => {
     nsp.to(rooms).emit('comments', event)
   })
 
-  const { pollIntervalSeconds, syncThrottleSeconds } = config.gitHub
-  const syncManager = new SyncManager(loadOpenIssuesAndComments, {
+  const { pollIntervalSeconds, syncThrottleSeconds, syncConcurrency } = config.gitHub
```

Comment:
```js
const { 
  pollIntervalSeconds = 0, 
  syncThrottleSeconds = 0, 
  syncConcurrency = 10 
  } = config.gitHub
  const syncManager = new SyncManager(() =>loadOpenIssuesAndComments(syncConcurrency), {
    interval: pollIntervalSeconds * 1000,
    throttle: syncThrottleSeconds * 1000,
    signal
  })
```
Why do you default the values when they are use and not when they are declared. Do you think it makes sense to have different defaults per usage?